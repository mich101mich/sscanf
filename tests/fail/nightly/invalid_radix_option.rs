fn main() {
    sscanf::sscanf!("", "{:#x#}", u8);
    sscanf::sscanf!("", "{:#r16}", u8);
    sscanf::sscanf!("", "{:r16#}", u8);
    sscanf::sscanf!("", "{:r}", u8);
    sscanf::sscanf!("", "{:r99}", u8);
    sscanf::sscanf!("", "{:r1}", u8);
    sscanf::sscanf!("", "{:x}", std::u8);
}
