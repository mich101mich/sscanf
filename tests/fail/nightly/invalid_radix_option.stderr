error: config modifier '#' can only be used with 'x', 'o' or 'b'
 --> tests/fail/nightly/invalid_radix_option.rs:2:28
  |
2 |     sscanf::sscanf!("", "{:#x#}", u8);
  |                            ^^^

error: config modifier '#' can only be used with 'x', 'o' or 'b'
 --> tests/fail/nightly/invalid_radix_option.rs:3:28
  |
3 |     sscanf::sscanf!("", "{:#r16}", u8);
  |                            ^^^^

error: config modifier '#' can only be used with 'x', 'o' or 'b'
 --> tests/fail/nightly/invalid_radix_option.rs:4:28
  |
4 |     sscanf::sscanf!("", "{:r16#}", u8);
  |                            ^^^^

error: radix option 'r' has to be followed by a number
 --> tests/fail/nightly/invalid_radix_option.rs:5:28
  |
5 |     sscanf::sscanf!("", "{:r}", u8);
  |                            ^

error: radix has to be a number between 2 and 36
 --> tests/fail/nightly/invalid_radix_option.rs:6:28
  |
6 |     sscanf::sscanf!("", "{:r99}", u8);
  |                            ^^^

error: radix has to be a number between 2 and 36
 --> tests/fail/nightly/invalid_radix_option.rs:7:28
  |
7 |     sscanf::sscanf!("", "{:r1}", u8);
  |                            ^^

error: radix options only work on primitive numbers from std with no path or alias
 --> tests/fail/nightly/invalid_radix_option.rs:8:33
  |
8 |     sscanf::sscanf!("", "{:x}", std::u8);
  |                                 ^^^^^^^
